.github/workflows/doc-build.yml

```
@@ -24,6 +24,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Maximize available disk space
+        uses: jlumbroso/free-disk-space@main
```

```suggestion
        uses: jlumbroso/free-disk-space@v1.3.1
```

It's probably safer to pin the version.

---

OK.